test/core/rename/Rename_test.cpp

```diff
@@ -341,25 +372,70 @@ void runTest(int argc, char **argv)
 
     sparta_assert(false == datafiles.empty(), "Need an output file as the last argument of the test");
 
-    sparta::Scheduler sched;
-    RenameSim sim(&sched, "mavis_isa_files", "arch/isa_json", datafiles[0], input_file);
+    if(input_file == "raw_integer.json"){
+        sparta::Scheduler scheduler;
+        uint64_t ilimit = 0;
+        uint32_t num_cores = 1;
+        bool show_factories = false;
+        OlympiaSim sim("simple",
+                       scheduler,
+                       num_cores, // cores
+                       input_file,
+                       ilimit,
+                       show_factories);
+        cls.populateSimulation(&sim);
+        sparta::RootTreeNode* root_node = sim.getRoot();
+
+        olympia::ExecutePipe* my_executepipe = root_node->getChild("cpu.core0.execute.alu0")->getResourceAs<olympia::ExecutePipe*>();
+        olympia::ExecutePipe* my_executepipe1 = root_node->getChild("cpu.core0.execute.alu1")->getResourceAs<olympia::ExecutePipe*>();
+        olympia::ExecutePipeTester executepipe_tester;
+        cls.runSimulator(&sim, 7);
+        executepipe_tester.test_dependent_integer_first_instruction(*my_executepipe);
+        executepipe_tester.test_dependent_integer_second_instruction(*my_executepipe1);        
+    }
+    else if(input_file == "raw_lsu.json"){
+        sparta::Scheduler scheduler;
+        uint64_t ilimit = 0;
+        uint32_t num_cores = 1;
+        bool show_factories = false;
+        OlympiaSim sim("simple",
+                       scheduler,
+                       num_cores, // cores
+                       input_file,
+                       ilimit,
+                       show_factories);
+        cls.populateSimulation(&sim);
+        sparta::RootTreeNode* root_node = sim.getRoot();
```

Comment:
This code is the same regardless of the input_file.  Can probably be moved outside of the `if-else`

Reply:
Addressed in [4fd928d](https://github.com/riscv-software-src/riscv-perf-model/pull/51/commits/4fd928d4400af49de00ac3f5f8b5c4990bdbb2fd)